gridstatus/isone.py

```diff
@@ -858,6 +858,116 @@ def _hour_to_interval_index(hour):
 
         return selected_intervals
 
+    @support_date_range(frequency=None)
+    def get_reserve_zone_prices_designations_real_time_5_min_final(
+        self,
+        date,
+        end=None,
+        verbose=False,
+    ):
+        """Return final five-minute reserve zone requirements, prices, and designations
+
+        Published and sometimes updated in the days following the operating day.
+
+        Args:
+            date: Date to query. Supports "latest" and "today"
+            end: End date for date range queries
+            verbose: Enable verbose logging
+
+        Returns:
+            DataFrame with columns: Interval Start, Interval End, Reserve Zone ID,
+            Reserve Zone Name, Ten Min Spin Requirement, Ten Min Requirement,
+            Total Requirement, TMSR Designated MW, TMNSR Designated MW,
+            TMOR Designated MW, TMSR Clearing Price, TMR Clearing Price,
+            Total Reserve Clearing Price
+        """
+        if date == "latest":
+            # Try today first then yesterday then two days ago
+            df = self.get_reserve_zone_prices_designations_real_time_5_min_final(
+                "today",
+                verbose=verbose,
+            )
+            if df.empty:
+                df = self.get_reserve_zone_prices_designations_real_time_5_min_final(
+                    pd.Timestamp.now(tz=self.default_timezone).normalize()
+                    - pd.Timedelta(days=1),
+                    verbose=verbose,
+                )
+                if df.empty:
+                    df = (
+                        self.get_reserve_zone_prices_designations_real_time_5_min_final(
+                            pd.Timestamp.now(tz=self.default_timezone).normalize()
+                            - pd.Timedelta(days=2),
+                            verbose=verbose,
+                        )
+                    )
+
+            return df
+
+        date_str = date.strftime("%Y%m%d")
+
+        # If end is provided, query the range; otherwise query single day
+        if end is not None:
+            end_str = end.strftime("%Y%m%d")
+            url = f"https://www.iso-ne.com/transform/csv/fiveminreserveprice?type=final&start={date_str}&end={end_str}"
+        else:
+            url = f"https://www.iso-ne.com/transform/csv/fiveminreserveprice?type=final&start={date_str}&end={date_str}"
```

Comment:
I'm surprised to see the whole hardcoded URL here, I can't recall if this is our convention however